### ISSUE TITLE
Missing author.url in ArticleJsonLd

### ISSUE THREAD

**Issue by apang2013** [garmeeh/next-seo#924]
## Problem

In the guidelines for writing structured data for [Articles](https://developers.google.com/search/docs/advanced/structured-data/article), the `author` object includes a `name` and a `url` property. The `url` is supposed to be the link to the page of the author who wrote the article.

Currently (5.0.0), in the `ArticleJsonLd` component, only `authorName` is provided, and does not provide a property where authors' urls can be applied.

## Current Behavior
Only authors' names can be specified:
```javascript
<ArticleJsonLd
  authorName: [ "Richard Morrow" ]
/>
```
we get (from Rich Results Test):
![image](https://user-images.githubusercontent.com/18233589/151752595-5e21b54a-59cf-4d24-8a9d-d0dfedc540b2.png)

## Suggestion

Change `ArticleJsonLd.authorName` to `ArticleJsonLd.authors`
where `authors` is an array of `author` with a signature like: 
```javascript
author: {
  name: string
  url: string
}
```
So that we can specify the url's of the authors of the article.
Usage would be like:
```javascript
<ArticleJsonLd
  authors: [
    {
      name: "Richard Morrow",
      url: "..."
    }
  ]
/>
```

**Comment by devanonymous** [comment]
there is a solution?

**Comment by goforbg** [comment]
Same here, is there any solution?

**Comment by apang2013** [comment]
Seems this is solved in #961, marking this as completed.
